aws-sdk-core/lib/aws-sdk-core/ini_parser.rb

```diff
@@ -0,0 +1,77 @@
+module Aws
+  class IniParser
+
```

Comment:
Given there are three class level methods, I would move them inside the eigenclass:

```ruby
class << self
  def ini_parse(raw)
  end
  # ...
end
```